packages/trip-details/src/index.tsx

```diff
@@ -0,0 +1,247 @@
+import flatten from "flat";
+// @ts-expect-error FIXME: Create TypeScript types for core-utils packages.
```

Comment:
I would personally push for the approach used in #262 of putting the message in package.json so it's harder to forget about, but am sway-able

Reply:
I added the message to the `tsc` script of the trip-details package, in addition to the TypeScript exception comment which is required or a compilation error occurs. (see https://github.com/opentripplanner/otp-ui/pull/281/commits/d39e81032da5df5266a7dfd0e664940c9cdbdd5f)

Reply:
Really like this approach thanks!